File: testdata/DefaultConstructorWithBody.java
class DefaultConstructorWithBody
{
  public DefaultConstructorWithBody()
  {
    int i = 0;
    while (i < 3)
    {
      state[i] = "" + i;

      if (i == 2)
      {
        System.out.println("Almost done");
      }

      i++;
    }
  }

  String[] state = new String[3];
}

File: testdata/expected/default/DefaultConstructorWithBody.scala
class DefaultConstructorWithBody
{
  {
    var i : Int = 0
    while (i < 3)
    {
      state[i] = "" + i

      if (i == 2)
      {
        System.out.println("Almost done")
      }

      i++
    }
  }

  var state : Array[String] = new Array[String](3)
}


